docs/modules/mirtop.md

```diff
@@ -0,0 +1,13 @@
+---
+Name: mirtop 
```

Comment:
With capitalisation? eg. `miRTop`? That's ok here - this is used to generate the MultiQC website.